_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,299 @@
+---
+title: 'Bitcoin Optech Newsletter #100'
+permalink: /en/newsletters/2020/06/03/
+name: 2020-06-03-newsletter
+slug: 2020-06-03-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a proposed design for a coinswap
+implementation, describes new middleware for allowing lightweight
+wallets to get information directly from a user's own node, and links to
+two transaction size calculators.  Also included are our regular
+sections with descriptions of several recently transcribed talks, new
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.  A special final section celebrates the
+publication of Newsletter #100.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Design for a coinswap implementation:** Chris Belcher
+  [posted][belcher coinswap] a design for a full-featured coinswap
+  implementation.  Coinswap is a protocol that allows two users to
+  create a pair of transaction that look like regular payments but which
+  actually swap their coins with each other.  This improves the privacy
+  of not just the coinswap users but all Bitcoin users, as anything that
+  looks like a payment could have instead been a coinswap.
+
+    Belcher's post summarizes the history of the coinswap idea, suggests
+    ways the multisig conditions needed for coinswap could be disguised
+    as more common transaction types, proposes using a market for
+    liquidity (like JoinMarket already does), describes splitting and
+    routing techniques to reduce privacy losses from amount correlation
+    or spying participants, mentions alternative coinswap protocols such
+    as succinct atomic swaps (see [Newsletter #98][news98 sas]), suggests
+    combining coinswap with [payjoin][topic payjoin], and discusses some
+    of the backend requirements for the system.  Additionally,
+    comparisons are made to other privacy techniques such as using LN,
```

Comment:
nit: perhaps active voice > passive voice here

Reply:
A lot to unpack; maybe worth testing a separation of this paragraph into three:

(1) the post begins by...
(2) the post then goes on to compare...
(3) near the end...

Reply:
I think it's ok as one paragraph; the goal of the paragraph isn't to teach people anything but simply to provide a large number reasons that they should go read the original email.  If it feels like there's a lot to unpack, then I think that means it's mission accomplished.  :-)